extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -288,10 +288,12 @@ export class SchemaCompareDialog {
 		if (isTarget) {
 			this.targetFileButton = view.modelBuilder.button().withProperties({
 				label: '•••',
```

Comment:
Probs a good opportunity to add a tooltip, too, with the same text as the ariaLabel